From c167bde3c0ce9bc602eece89908d29d350f2c6be Mon Sep 17 00:00:00 2001
Message-Id: <patch-01.17-c167bde3c0c-20230317T152724Z-avarab@gmail.com>
In-Reply-To: <cover-00.17-00000000000-20230317T152724Z-avarab@gmail.com>
References: <cover-00.17-00000000000-20230317T152724Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 6 Dec 2022 01:24:00 +0100
Subject: [PATCH 01/17] cocci: remove dead rule from
 "the_repository.pending.cocci"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The "parse_commit_gently" macro went away in [1], so we don't need to
carry his for its migration.

1. ea3f7e598c8 (revision: use repository from rev_info when parsing
   commits, 2020-06-23)

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 contrib/coccinelle/the_repository.pending.cocci | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/contrib/coccinelle/the_repository.pending.cocci b/contrib/coccinelle/the_repository.pending.cocci
index 747d382ff5f..23b97536da5 100644
--- a/contrib/coccinelle/the_repository.pending.cocci
+++ b/contrib/coccinelle/the_repository.pending.cocci
@@ -34,14 +34,6 @@ expression G;
 + repo_parse_commit_internal(the_repository,
   E, F, G)
 
-@@
-expression E;
-expression F;
-@@
-- parse_commit_gently(
-+ repo_parse_commit_gently(the_repository,
-  E, F)
-
 @@
 expression E;
 @@
-- 
2.40.0.rc1.1034.g5867a1b10c5

